traffic_ops/traffic_ops_golang/cdn/domains.go

```diff
@@ -0,0 +1,97 @@
+package cdn
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/apache/trafficcontrol/lib/go-tc/v13"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/jmoiron/sqlx"
+)
+
+// GetDomainsList gathers a list of domains, except for the domain name.
+// There seems to be an issue nesting queries (performing a query while
+// rows.Next still needs to iterate). https://github.com/lib/pq/issues/81
+func getDomainsList(tx *sqlx.Tx) ([]v13.Domain, []int, error) {
+
+	var (
+		cdn  int
+		id   int
+		name string
+		desc string
+	)
+
+	domains := []v13.Domain{}
+	cdn_ids := []int{}
+
+	q := `SELECT cdn, id, name, description FROM Profile WHERE name LIKE 'CCR%'`
```

Comment:
I found a const variable declared `RouterType` in monitoring.go that is equal to "CCR". Does that work too?

Reply:
It would, but monitoring is in the `main` package, you can't import it here. 

The `monitoring.go` needs moved to its own package, and `CCR` needs put in `go-tc/enum.go` for everything to use. I have PRs for both, but they haven't been merged yet.

Reply:
Ok, good to know.